### ISSUE TITLE
make tasks to improve calculate_mean_damage

### ISSUE THREAD

**Issue by diegoac2** [IN-CORE/pyincore#12]
Many more improvements can be made to "calculate_mean_damage()" method so it is not very rigid.
https://opensource.ncsa.illinois.edu/bitbucket/projects/INCORE1/repos/pyincore/browse/pyincore/utils/analysisutil.py#95
Some are: # remove the 2nd line of the csv that says "java.lang.string,..." etc. incore doesn't need it anymore and is causing us to write workarounds on incore-ui and pyincore

Remove references to Slight, moderate etc. references in the CSVs and also in this method.
The code still refers to the csv lines by indexes, and it will break if the no of the damage states change(to say 3 or 6). It should work with DS values as keys instead of indexes
Standardize the column names. Some use "Best Mean Damage Ratio" and others "Mean Damage Ratio". These are conditionally hardcoded
Bottom line is, this method is very fragile and needs some refactoring after discussion with someone who understands the science and terminology behind it.

**Comment by navarroc** [comment]
I agree, we should clean this up and I can help with some of it. The current implementation of mean damage only supports 3 damage states for building structural and non structural damage and 4 damage states for bridges. Anything outside of that range is not supported and would need to be referred the researchers.

**Comment by navarroc** [comment]
I've added two tasks to fix issues I've found with mean damage. One task is around harmonizing the column names for the different mean damage ratio datasets, added a column to map the Ergo damage states to our DS_ keys, and fixing the internal code references. The other is around checking to make sure the data type has a supported mean damage factor and that the number of damage interval keys matches the data type and we should throw an unsupported exception if it is different.
